internal/lib/lib.go

```diff
@@ -1936,6 +1936,18 @@ func GetIPFromNode(node *v1.Node) (string, string) {
 				nodeV6 = strings.Split(nodeIP, "/")[0]
 			}
 		}
+		if GetCNIPlugin() == ANTREA_CNI {
+			if nodeIPstr, ok := node.Annotations["node.antrea.io/transport-addresses"]; ok {
```

Comment:
@aaha97 : Where can i get information about this annotation? will it be always used to indicate IPV4, V6  node ip?